docs/data/_index.md

```diff
@@ -19,23 +19,82 @@ menuindent: true
 # SME: Alexa Greenberg
 ---
 
-Viam's built-in data management service allows you to capture, sync, [view](view/), [label](dataset/), and [export](export/) data from your machines, as well as use [Machine Learning](/ml/) features to enhance your machine's capabilities.
-The service is designed for flexibility and efficiency while preventing data loss, data duplication, and other data management issues.
+The data management service is a robust solution for smart machine data handling, [data capture](/data/capture/), and [cloud synchronization](/data/cloud-sync/).
+Using the data management service, you can collect data from different parts of a robot, IoT device, or any other machine.
+Then, you can configure which data to sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine.
+The data management service also keeps your data secure during transfer.
+
+## Overview
 
 ![Data is captured on the machine, uploaded to the cloud, and then deleted off local storage.](/data/data_management.png)
 
-The service has two parts: [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/).
+Viam's data management features can be broken down into on-device and cloud data capabilities:
+
+### On-device data management
 
-Once you have captured and synced data, you can:
+Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
 
 {{< cards >}}
-{{% card link="/data/view/" %}}
-{{% card link="/data/dataset/" %}}
-{{% card link="/data/export/" %}}
-{{% card link="/data/query/" %}}
+{{% manualcard link="/data/capture/" %}}
+
+#### Data capture
+
+Configurable for various robot components like cameras, sensors, and motors; set capture frequency, with automatic resumption after robot restarts.
+
+{{% /manualcard %}}
+{{% manualcard link="/data/cloud-sync/" %}}
+
+#### Cloud sync
+
+Securely transfer data to cloud at the frequency you define, resilient to interruptions, and delete local data post-sync for space management.
+
+{{% /manualcard %}}
 {{< /cards >}}
 
-## Used with
+### Cloud data management
+
+Streamlined data handling with advanced querying, viewing, and filtering capabilities, along with efficient data labeling and exporting tools.
```

Comment:
TODO (comment got outdated): reframe descriptions here to speak to user more than just highlight capabilities